scripts/badger/process_bribes_graviaura.py

```diff
@@ -121,15 +122,35 @@ def sell_weth(badger_total="0"):
         PROCESSOR.swapWethForBadger(order_payload, order_uid)
 
     if aura_share > 0:
-        order_payload, order_uid = SAFE.badger.get_order_for_processor(
-            PROCESSOR,
-            sell_token=WETH,
-            mantissa_sell=aura_share,
-            buy_token=AURA,
-            deadline=DEADLINE,
-            coef=COEF,
-            prod=COW_PROD,
-        )
+        # Check which path is better
+        #  1. Swap WETH for AURA and deposit to GRAVI_AURA
+        #  2. Swap WETH directly for GRAVI_AURA
+        buy_amount_aura = SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share)['buyAmountAfterFee']
```

Comment:
from the api, the come back in str format

```suggestion
        buy_amount_aura = int(SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share)['buyAmountAfterFee'])
```